README.md

```diff
@@ -91,9 +104,18 @@ Reference:
   - `generator`, _(optional)_ use a generator for the value, currently only support `secret`
 - `options`: _(optional)_ Options when deploying the service
   - `allow-unauthenticated`: _(optional, default: `true`)_ allow unauthenticated requests
+- `build`: _(optional)_ Build configuration
+  - `skip`: _(optional, default: `false`)_ skips the built-in build methods (`docker build`, `Maven Jib`, and `buildpacks`)
 - `hooks`: _(optional)_ Run commands in separate bash shells with the environment variables configured for the
- application and environment variables for `GOOGLE_CLOUD_PROJECT`, `GOOGLE_CLOUD_REGION`, and `K_SERVICE`. Command
- outputs are shown as they are executed.
+ application and environment variables `GOOGLE_CLOUD_PROJECT` (Google Cloud project), `GOOGLE_CLOUD_REGION`
+ (selected Google Cloud Region), `K_SERVICE` (Cloud Run service name), `IMAGE_URL` (container image URL), `APP_DIR`
+ (application directory). Command outputs are shown as they are executed.
+  - `prebuild`: _(optional)_ Runs the specified commands before running the built-in build methods. Use the `IMAGE_URL`
+  environment variable for the container storage coordinates.
+    - `commands`: _(array of strings)_ The list of commands to run
+  - `postbuild`: _(optional)_ Runs the specified commands after running the built-in build methods. Use the `IMAGE_URL`
```

Comment:
> Use the `IMAGE_URL`
  environment variable for the container storage coordinates

the "coordinates" and "container storage" sound odd, never heard in this context.

What if we said:

> Use `IMAGE_URL` to determine the container image name you need to build and push.